x/programs/rust/examples/counter/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-use wasmlanche_sdk::{program::Program, public, state_keys, types::Address};
+use wasmlanche_sdk::{program::Program, public, state_keys, types::Address, state::from_raw_ptr};
```

Comment:
the macro requires this function, is there a way to use this in `sdk_macros` without a dependency cycle?